29_sep_2022_network_certificate_part_3/practice_part3.md

```diff
@@ -0,0 +1,31 @@
+## Practice Generate self signed SSL Certificate
+	
+1. Prerequisites
+		
+Install python3 (Rocky)
+
+```
+sudo dnf install python3 -y
+```
+
+python3 is installed to manjaro. To run python script in manjaro use command
```

Comment:
python3 is already installed on Manjaro  
please rename manjaro to Manjaro throughout the text